src/react-components/src/__tests__/components/DateRangeForm-test.tsx

```diff
@@ -0,0 +1,39 @@
+
+import { expect } from "chai";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+import * as Mocha from "mocha";
+import { stub } from "sinon";
+
+import DateRangeForm from "../../components/01-atoms/Forms/DateRangeForm";
+
+describe("Renders Input", () => {
+  let container;
+  let callback;
+  before(() => {
+    callback = stub();
+    const container = Enzyme.mount(<DateRangeForm
+      formLabel={<>Publication Year</>}
+
+      fromLabelOpts={{ labelContent: <>From</>, id: "FromLabel" }}
+      fromInputOpts={{ inputId: "fromInput" }}
+      fromHelper={{ content: <>EX. 1901</>, id: "fromyearhelper", isError: false }}
```

Comment:
I was debating whether or not to even leave this comment, but here goes anyway: the designs _do_ have this string exactly as you've mocked it up here, but I've never seen "example" abbreviated this way. I'd love it if in our stories at least if we could mock it up as, "E.x., 1901"